src/__tests__/utils.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { bem } from '../utils';
+
+describe('Utils', () => {
+  const cx = bem('testWidget');
+
+  describe('bem', () => {
+    it('generates className for root', () => {
+      expect(cx()).toBe('ais-testWidget');
+    });
+
+    it('generates className for an element', () => {
+      expect(cx('input')).toBe('ais-testWidget-input');
+    });
+
+    it('generates className for header, body and footer', () => {
```

Comment:
Those cases are never used AFAIK (I searched in the source code) and it seems like they're too specific to be part of the `bem` helper. I think they can be removed.

Reply:
Hmm I just found `<div [class]="cx('body')">` in `range-slider.ts` which generates a class name `ais-RangeSlider-body ais-body`. Not sure `ais-body` is actually used. At least I cannot find it from the source code. But if any user is using it to style, removing it will be a breaking change. What do you think?

Reply:
Ah, I didn't find this occurrence. V3 is still in beta so this is the right time to remove it because this is not part of the [RangeSlider spec](https://instantsearch-css.netlify.com/widgets/range-slider/).

Reply:
The spec doesn't even have a div with `ais-RangeSlider-body`, but I'm not sure how much I can trust with the RangeSlider spec, because the implementation differs from flavors. 
Even the spec says

```html
<div class="ais-RangeSlider">
  <div class="rheostat rheostat-horizontal" style="position: relative;">
```

but from the storybook of React-IS, the inspector shows

```html
<div class="ais-RangeSlider rheostat rheostat-horizontal" style="position: relative;">
  <div class="rheostat-background"></div>
  <button ...
```

So for now, I'll just remove `ais-body` from the component and the utils. And removing the div with `ais-RangeSlider-body` seems to be a different subject.

Reply:
Yes, we'll reiterate on the component before stable release.

Reply:
Okay! 👍

Reply:
62b1636